enarx-keep-sgx-shim/src/event.rs

```diff
@@ -26,7 +31,7 @@ pub extern "C" fn event(
         Some(Exception::InvalidOpcode) => {
             match unsafe { core::slice::from_raw_parts(h.aex.gpr.rip as *const u8, 2) } {
                 // syscall
-                [0x0f, 0x05] => {
+                OP_SYSCALL => {
```

Comment:
I think the constants provided by your patch make the comment above redundant. :slightly_smiling_face: 

:+1: for readability

Reply:
Comment removed